runtime/thunk/thunk.ts

```diff
@@ -307,7 +308,13 @@ async function doFindAndExecutePackFunction<T extends FormulaSpecification>({
           }
 
           break;
-
+        case MetadataFormulaType.ValidateParameters: {
```

Comment:
Before running a formula/sync, should we run the validateParameters metadata formula in here? Or only do that in the coda repo

Reply:
I don't think we should auto-run it in the SDK, let's let the `coda` side orchestrate that, gives us more control.